code/online_challenges/src/project_euler/problem_004/problem_004.java

```diff
@@ -0,0 +1,30 @@
+    public class problem_004 {
+
+      public static void main( String[] args) {
+
+               int prod,temp,sum,rem,lar = 0,i,j;
```

Comment:
Add space after comma.

Reply:
Rather then declaring i and j variables here, you can declare variables in the for loop as for(int i = 100; i<= 999; ++i), same for j.